src/sinks/amqp/sink.rs

```diff
@@ -0,0 +1,139 @@
+use crate::{
```

Comment:
IDK if we want to adopt it as a convention but I kind of find it useful to have a docstring at the top of a file like this which gives a little overview/background info of the integration. Can also call out if any docker-compose files / make targets invoke the integration tests.

(or the mod.rs file :shrug: )